src/ui/Card.tsx

```diff
@@ -48,11 +47,10 @@ function randomSign(): number {
 }
 
 export function Card(props: CardProps): JSX.Element {
-  const [messages, setMessages] = useState([] as Message[]);
-  const [
-    firstAndLastMessageContents,
-    setFirstAndLastMessageContents,
-  ] = useState([] as Message[]);
+  const messages: Message[] = props.thread.messages;
+  const firstAndLastMessages = props.thread.firstAndLastMessages
```

Comment:
Ditto comment above would we get here without these having been fetched? If it is actually possible then this has a bug in Card I think since Card won't know to rerender when we finally fetch them.